pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -69,6 +69,7 @@ RUN apt-get update \
     build-essential \
     ca-certificates \
     cmake \
+    libgssapi-krb5-2 \
```

Comment:
Please revert this change. DLC team will handle the patching of this image and release it soon, thereafterJumpstart DLCs can consume the latest images without the code change.

Reply:
Hi @shantanutrip , is any other images that DLC teams are patching together? Asking because there are other PRs inflight and we want to avoid duplicate works.